static/bootstrap-3.3.4-dist/css/bootstrap-theme.css

```diff
@@ -0,0 +1,476 @@
+/*!
+ * Bootstrap v3.3.4 (http://getbootstrap.com)
+ * Copyright 2011-2015 Twitter, Inc.
+ * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
+ */
+
+.btn-default,
```

Comment:
you can probably host bootstrap through a CDN, but it is OK to have it in your repo